storage/client/client.go

```diff
@@ -85,6 +94,46 @@ func wrapError(err error) error {
 	return apiCommon.ErrStorageError{Err: err}
 }
 
+// Returns the rows for a given query as well as the total count of matching records
+// for queries that return multiple rows.
```

Comment:
```suggestion
// For queries that return multiple rows, returns the rows for a given query, as well as
// the total count of matching records, i.e. the number of rows the query would return
// with limit=infinity.
```

(Putting "For queries" first to make it clear which part of the sentence it refers to; explaining how "total count" is different from len(rows))

I think it would be useful to be similarly explicit/wordy in openapi because it's a subtle distinction.

Reply:
Nice, updated!